lib/text_styles/custom_text_style.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+import '../ui_constants.dart';
+
+class CustomTextStyle {
+  static const TextStyle customTextStyleBoldWhite20 = TextStyle(
+    fontWeight: FontWeight.w900,
+    fontSize: UiConstants.fontSize20,
+    color: Colors.white70,
+  );
+  static const TextStyle customTextStyleWhite = TextStyle(
+    color: Colors.white,
+  );
+  static const TextStyle customTextStyleBoldWhite25 = TextStyle(
+    fontWeight: FontWeight.bold,
+    fontSize: UiConstants.fontSize25,
+    color: Colors.white,
+  );
+  static const TextStyle customTextStyleBold20 = TextStyle(
+    fontSize: UiConstants.fontSize20,
+    fontWeight: FontWeight.bold,
+  );
```

Comment:
Well done with this! A small comment:
As the class is already named `CustomTextStyle`, there's no need to add that prefix to each inner field!

Reply:
done